README.md

```diff
@@ -1,2 +1,15 @@
 # community-protocols
-Cross-component coordination protocols
+As the diversity of custom elements grows wider and deeper, so does the benefits of those elements being able to communicate with each other via reusably well-known protocols that ensure components from X vendor can interoperate with components from Y vendor. This project looks to define community protocols for cross-component coordination that reduce the overhead required to make that possible without the side effect of locking consumers of those elements into specific patterns themselves.
+
+Think of things like a context API, async work (Suspense-like), SSR-interop, incremental hydration, etc., many of these capabilities get locked into framework APIs that increase the barriers to interoperability. With a community defined protocol these features and more could become more plug-n-play, more opt-in, and more individually susseptible to the sort of competition that drives innovation and change for the better on the web. 
```

Comment:
The final sentence of this paragraph could use some rework I think. The word "more" is used a bit much. Also "plug-n-play" and "opt-in" seem to be getting at the same core idea. Perhaps that can be made more succinct. Finally, the final clause of the last sentence appears to imply that conformance is a catalyst for innovation, which I don't find much evidence for. Perhaps I've misunderstood the idea there. I think that should be reworded or perhaps dropped.

Reply:
Makes sense.